PR CL-76 Create Integration tests for StockPortfolioController

SemBen opened the pull request:
Complete integration tests for StockPortfolioController

sinerger commented on CRMLite.TransactionStoreCore/Entities/StockPortfolio.cs:
```diff
@@ -7,7 +7,7 @@ public class StockPortfolio
         public Guid ID { get; set; }
         public Guid LeadID { get; set; }
-        public Stock Stocks { get; set; }
+        public Guid StockID { get; set; }
```
возможно есть смысл тут хранить именнно акцию ,а не ее айди

sinerger commented on CRMLite.TransactionStoreDatabase/CRMLite/StoredProcedures/StockPortfolio/GetAllStockPortfoliosByLeadID.sql:
```diff
@@ -1,11 +1,11 @@
+AS
+SELECT SP.ID,
+	SP.LeadID,
+	S.ID AS StockID,
```
к комменту выше, хранимка возвращет тебе полностью все что нужно для того что бы хранить акцию а не ее айди. исправь на акцию. тут учти порядок в селекте

SemBen replied:
что не так с порядком?

SemBen committed:
Complete integration tests for StockPortfolioController


Delete project


Fix Bug


Test app veyor


Create GetWalletByID testMethod


Complete integration tests for StockPortfolioController


Refactor


Complete task
